integration-tests/terraform/common/output.tf

```diff
@@ -0,0 +1,27 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+output "testing_id" {
+  value = random_id.testing_id.hex
+}
+
+output "cwa_iam_role" {
+  value = "cwa-e2e-iam-role"
```

Comment:
sounds like the name for most of these should reference the operator to differentiate resources for the agent.

ie. instead of `cwa-e2e-iam-role`, use `cwa-operator-e2e-iam-role`

Reply:
Will try to do this as part of EKS beta cluster creation PR